### ISSUE TITLE
Move Fenix Nightly to "org.mozilla.fenix.nightly" GPS app

### ISSUE THREAD

**Issue by mitchhentges** [mozilla-mobile/fenix#2881]
Currently, the Fenix nightly is occupying the "Fenix Release" (`org.mozilla.fenix`) slot in Google Play.
We need to move Fenix Nightly to `org.mozilla.fenix.nightly` so that we're ready for the real release.

* [x] This depends on [this](https://github.com/mozilla-releng/pushapkscript/pull/87) landing

Todo:

* [x] Continue publishing nightly to `org.mozilla.fenix` on the `nightly` channel to support existing users (handled by [this PR](https://github.com/mozilla-mobile/fenix/pull/3225))
* [x] Additionally, publish nightly to `org.mozilla.fenix.nightly` on the `beta` channel

**Comment by mitchhentges** [comment]
Sylvestre and I will set this up on Monday. The plan is:

1. Sylvestre creates and passes me `fenix-production@boxwood-...` for deploying to `org.mozilla.fenix`
2. I update existing deployments of production and nightly (the nightly deploying to `org.mozilla.fenix` on the `nightly` track) to use the new creds
3. Sylvestre removes `fenix-nightly@boxwood-...`
4. Sylvestre creates and passes me `fenix-nightly@boxwood-...` for deploying to `org.mozilla.fenix.nightly`
5. I configure nightly automation to continue publishing to `org.mozilla.fenix@nightly` in addition to pubishing to `org.mozilla.fenix.nightly@beta`

**Comment by mitchhentges** [comment]
We've learned that we can't keep publishing a nightly to the production app's `nightly` track: users don't stay on the track they've chosen. Rather, they're always updated to the most up-to-date app that is equal to or more stable than the track they're on.

So, rather than this:
```
[prod user]:    1.0            -> 1.1                     -> 2.0
[nightly user]: Jan 1 -> Jan 2 -> Jan 3 -> Jan 4 -> Jan 5 -> Jan 6
```
Instead, the nightly user might be upgraded to a production version, then back to a nightly:
```
[prod user]:    1.0                   -> 1.1                            -> 2.0
[nightly user]: 1.0 -> Jan 1 -> Jan 2 -> 1.1 -> Jan 3 -> Jan 4 -> Jan 5 -> 2.0 -> Jan 6
```

We don't want nightly users having production apps installed due to potential unexpected database/state issues (what if a release hotfix gets a database migration that `nightly` isn't supposed to have yet?) as well as branding (users might be confused why their Nightly Fenix icon shows production Fenix).

So, we'll need to stop publishing Fenix nightlies on the `org.mozilla.fenix` `nightly` channel, and instead move it to `org.mozilla.fenix.nightly` where it won't interact in weird ways.

Julien and I have updated production fenix to use `fenix-production@boxwood...` instead of `fenix-nightly@boxwood` credentials today, so hopefully soon we'll have fenix nightly push credentials so we can get the new nightly up and running
